integration-cli/docker_cli_network_unix_test.go

```diff
@@ -507,6 +507,19 @@ func (s *DockerSuite) TestDockerInspectMultipleNetwork(c *check.C) {
 		Err:      "Error: No such network: nonexistent",
 		Out:      "[]",
 	})
+
+	// Should not fail fast, and still print host network but print an error
+	result = dockerCmdWithResult("network", "inspect", "nonexistent", "host")
+	c.Assert(result, icmd.Matches, icmd.Expected{
+		ExitCode: 1,
+		Err:      "Error: No such network: nonexistent",
+		Out:      "host",
+	})
+
+	networkResources = []types.NetworkResource{}
+	err = json.Unmarshal([]byte(result.Stdout()), &networkResources)
+	c.Assert(err, check.IsNil)
+	c.Assert(networkResources, checker.HasLen, 1)
```

Comment:
Yes, it has been updated. Since it has been moved into a new test func `TestDockerInspectMultipleNetworksIncludingNonexistent `, but the content keeps the same.